### PR TITLE
:sparkles: Expose technical api

### PR DESCRIPTION
**Pull request by mickaelalibert**
## What type of PR is this?
Improvement in configurability

## What this PR does / why we need it:

This PR allows to expose the [technical api](https://docs.gravitee.io/apim_installguide_management_api_technical_api.html) of the management app for monitoring purposes

## Which issue(s) this PR fixes:

None

### CODE REVIEW

File: apim/templates/api-configmap.yaml [hunk 1]
```diff
@@ -48,14 +48,15 @@ data:
     services:
       core:
         http:
-          enabled: true
-          port: 18083
-          host: localhost
+          enabled: {{ .Values.api.http.services.core.http.enabled }}
```
Comment (brasseld) [inline apim/templates/api-configmap.yaml]: You may probably have to manage the default value to `true` for those who are not using the default `values.yml` and providing their own.

Comment (brasseld) [reply apim/templates/api-configmap.yaml]: (I mean, for backward compatibility)

Comment (mickaelalibert) [reply apim/templates/api-configmap.yaml]: I removed the possibility to disabled the technical api, as this is not really needed, and testing a value to be nil/false/true in helm isn't an easy task (cf multiple issues as this one https://github.com/helm/helm/issues/3308)

**Comment by brasseld** [comment]
LGTM.

Please squash your 4 commits into a single one.

**Pull request merged**